plugins/pubIds/doi/templates/settingsForm.tpl

```diff
@@ -63,6 +63,14 @@
 			{include file="linkAction/linkAction.tpl" action=$clearPubIdsLinkAction contextId="doiSettingsForm"}
 		{/fbvFormSection}
 	{/fbvFormArea}
+	{if ($enableIssueDoi || $enableSubmissionDoi || $enableRepresentationDoi) && $doiPrefix && $doiSuffix && $doiSuffix != 'customId' }
+		{fbvFormArea id="doiAssignJournalWideFormArea" title="plugins.pubIds.doi.manager.settings.doiAssignJournalWide"}
+			{fbvFormSection}
+				<span class="instruct">{translate key="plugins.pubIds.doi.manager.settings.doiAssignJournalWide.description"}</span><br/>
```

Comment:
Use a `<div>` instead of a `<span>` and you should be able to get rid of the `<br/>`.